python/cugraph/structure/number_map.py

```diff
@@ -96,7 +96,8 @@ def __init__(self, df, src_col_names, dst_col_names, id_type,
 
         def compute(self):
             if not self.numbered:
-                tmp = self.df.groupby(self.col_names).sum().sort_values(
+                tmp = self.df.groupby(
+                    self.col_names, sort=True).sum().sort_values(
```

Comment:
This shouldn't require `sort=True`.  The groupby is followed by a sort later on the line.